faust/models/record.py

```diff
@@ -592,7 +592,7 @@ def to_representation(self) -> Mapping[str, Any]:
         payload = self.asdict()
         options = self._options
         if options.include_metadata:
-            payload["__faust"] = {"ns": options.namespace}
+            payload[self._blessed_key] = {"ns": options.namespace}
```

Comment:
Apparently `options.allow_blessed_key` is defined in `faust/types/models.py`, do we want to add this as well to check before adding the blessed key?

Reply:
https://github.com/faust-streaming/faust/blob/ebc7777333885dccf846d99e0b72f07511b02092/faust/models/base.py#L153-L160

Guess we don't need the option here.